loc/en.js

```diff
@@ -60,22 +61,21 @@ module.exports = {
       registeredWallets: 'Registered wallets',
     },
     itIsAlive: {
-      subtitle: 'Airdrop is live! Check it out!',
-      description: 'We are happy to announce our first Bitcoin Vault Airdrop for Gold Wallet.',
-      iDontCare: "I don't care",
+      subtitle: 'Airdrop is live!',
+      description: 'Our first BTCV airdrop is here.\n Check it out to get rewards!',
```

Comment:
is the whitespace desired before the word `Check` in the new line?

Reply:
done